README.md

```diff
@@ -112,12 +112,6 @@ and make sure that the following sub-domains point to your Docker Swarm cluster
 
 **Note**: You can also use a subdomain, like `swarmprom.example.com`. Just make sure that the subdomains point to (at least one of) your cluster IPs. Or set up a wildcard subdomain (`*`).
 
-* Set and export an environment variable with the tag used by Traefik public to filter services (by default, it's `traefik-public`):
-
-```bash
-export TRAEFIK_PUBLIC_TAG=traefik-public
```

Comment:
I have never used this to override the Traefik tag, I don't know anyone that has. And I'm making it "fixed" by default, in DockerSwarm.rocks and in the project generators (for FastAPI), so it's easier to simplify that and remove the optional env var to reduce complexity/noise.